mmv1/templates/terraform/examples/network_security_server_tls_policy_basic.tf.erb

```diff
@@ -0,0 +1,22 @@
+resource "google_network_security_server_tls_policy" "<%= ctx[:primary_resource_id] %>" {
+  provider               = google-beta
+  name                   = "<%= ctx[:vars]['resource_name'] %>"
+  labels                 = {
+    foo = "bar"
+  }
+  description            = "my description"
+  allow_open             = "false"
+  server_certificate {
+    certificate_provider_instance {
+        plugin_instance = "google_cloud_private_spiffe"
+      }
+  }
+  mtls_policy {
+    client_validation_ca {
+      grpc_endpoint {
+        target_uri = "unix:mypath"
+      }
```

Comment:
`client_validation_ca` is an array in the schema, and a common issue we can often detect in acceptance tests is that APIs can reorder lists (so that it differs from the Terraform config's ordering) and this results in an unexpected diff detected by Terraform.

Could you please update this list to include multiple entries? That way the generated tests from this example could detect if this is an issue. It helps to try and make a 'messy' config that has entries out of alphabetical order and combined different types of elements (`certificate_provider_instance` and `grpc_endpoint`)